test/functional/lua/filetype_spec.lua

```diff
@@ -119,11 +119,11 @@ describe('vim.filetype', function()
 
   it('works with contents #22180', function()
     eq(
-      'sh',
+      'bash',
       exec_lua(function()
         -- Needs to be set so detect#sh doesn't fail
         vim.g.ft_ignore_pat = '\\.\\(Z\\|gz\\|bz2\\|zip\\|tgz\\)$'
-        return (vim.filetype.match({ contents = { '#!/usr/bin/env bash' } }))
+        return vim.filetype.match({ contents = { '#!/usr/bin/env bash' } })
```

Comment:
Because you changed this line. Change it back.
```suggestion
        return (vim.filetype.match({ contents = { '#!/usr/bin/env bash' } }))
```